### DISCUSSION TITLE
VS code problem with live server while installing node js and npm

### DISCUSSION THREAD

**Discussion by saravanars81** [coder/code-server#5847] [Q&A]
VS code problem  with first install the live server  it working but same time it's not working while install the node js   

 Follow to Change lot  YouTube but not what is  problem   I don't know 

Please tell solution.

**Comment by jsjoeio** [comment]
Can you share what steps you are doing?

**Reply by saravanars81** [reply]
My visual studio code  that problem with the Live server for purpose  web development HTML ,CSS run  but same time  i needed  Node Js  back-end to use that's problem  

How to install the node js externally in pc and install the live server    in extension to use  visual studio code 

While install node js  the live server extension not working in visual studio code 
 

Anyone tell me  correct procedure install and configuration to use it
@jsjoeio  @rjoydip11@coadler @bpmct @Leask

**Reply by jsjoeio** [reply]
My apologies, I don't fully understand. Can you provide a repro or a video to show what the problem is?
